# SPDX-FileCopyrightText: 2020-2021 Smart Robotics Lab, Imperial College London
# SPDX-FileCopyrightText: 2020-2021 Sotiris Papatheodorou
# SPDX-License-Identifier: CC0-1.0

# Simple MAV simulator settings
habitat_mav_sim:
    sim_freq: 60
    a_max: [1.0, 1.0, 0.5]
    w_max: [0.1, 0.1, 0.05]
    world_frame_id: "habitat"

